zerver/lib/bugdown/__init__.py

```diff
@@ -325,9 +338,22 @@ def __init__(self, md, bugdown):
         self.bugdown = bugdown
         markdown.treeprocessors.Treeprocessor.__init__(self, md)
 
+    def image_preview_enabled_for_realm(self):
+        # type: (Realm) -> bool
+        global current_message
+        if current_message is not None:
+            realm = current_message.get_realm()
+        else:
+            realm = None
+        if not settings.INLINE_IMAGE_PREVIEW:
+            return False
+        if realm is None:
+            return True
+        return realm.inline_image_preview
```

Comment:
This doesn't access self; can you move it to be a function (not a class method) next to the inline_url_embed_preview one?